feature/myPage/src/main/java/com/simhwa/mypage/MyPageViewModel.kt

```diff
@@ -0,0 +1,20 @@
+package com.simhwa.mypage
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.simhwa.data.mainpage.model.User
+import com.simhwa.data.mainpage.repository.MainPageRepository
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class MyPageViewModel @Inject constructor(private val repository: MainPageRepository) :
```

Comment:
별거 아니지만 가독성을 높이기 위해서 이렇게 나타내는게 좋을 것 같습니다

```suggestion
@HiltViewModel
class MyPageViewModel @Inject constructor(
    private val repository: MainPageRepository
) : ViewModel() {
    ...
}
```

Reply:
커맨드 쉬프트 L 누르는걸 까먹었네요😅